src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
```

Comment:
Create const variables for such magic values as 40, 20, 50
For example `const dailyCost = 40;`